src/background.js

```diff
@@ -1,5 +1,7 @@
 // keep an mru cache for every window
 
+var tabImages = []
```

Comment:
We should be storing this in chrome.storage.session or else it will get deleted after we unfocus Chrome